src/js/components/Tip/stories/Table.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import {
+  grommet,
+  Anchor,
+  Box,
+  DataTable,
+  Grommet,
+  Heading,
+  Text,
+  Tip,
+} from 'grommet';
+import { Trash, Refresh, Info } from 'grommet-icons';
+
+// Source code for the data can be found here
+// https://github.com/grommet/grommet/blob/master/src/js/components/DataTable/stories/data.js
+import { columns, DATA } from '../../DataTable/stories/data';
+
+const TipContent = ({ message, icon }) => (
+  <Box
+    background="dark-3"
+    round="xsmall"
+    pad="xsmall"
+    margin="xsmall"
+    direction="row"
+    gap="small"
+  >
+    {icon && <Info color="accent-4" />}
+    <Text color="accent-1">{message}</Text>
+  </Box>
+);
+
+const Example = () => {
+  const [data, setData] = useState(DATA);
+  const [selected, setSelected] = useState();
+
+  const removeRow = () => {
+    const filteredData = data.filter(item => item.name !== selected.name);
+    setData(filteredData);
+    setSelected(undefined);
+  };
+
+  return (
+    <Grommet full theme={grommet}>
+      <Box align="center" justify="center" background="dark-1" fill gap="large">
```

Comment:
Any reason so many have dark backgrounds? It seems inconsistent with most of the other stories.

Reply:
I just liked the contrast of the tooltip background in the dark mode. 
I'll flip a few examples to use light mode.

Reply:
I've removed the dark background from a few stories.